src/kOS/Suffixed/StageValues.cs

```diff
@@ -6,17 +6,21 @@
 using System.Collections.Generic;
 using System.Linq;
 using KSP.UI.Screens;
+using kOS.Module;
 
 namespace kOS.Suffixed
 {
     [kOS.Safe.Utilities.KOSNomenclature("Stage")]
     public class StageValues : Structure
     {
         private readonly SharedObjects shared;
+        private HashSet<global::Part> partHash = new HashSet<global::Part>();
+        private PartSet partSet;
```

Comment:
Yeah, this is their new more generic fix for removing active resources.  I like it too.  I'm not sure I see how we can integrate it with the name tag, but if you've got an idea I'm all for it.